ansible/roles/systemd/tasks/main.yml

```
@@ -0,0 +1,43 @@
+    dest: "{{ project_path }}/scripts/check_status.sh"
+    mode: '0755'
+    owner: "{{ project_user }}"
+    group: "{{ project_user }}"
+
+- name: Create application restart script
+  template:
+    src: restart_app.sh.j2
+    dest: "{{ project_path }}/scripts/restart_app.sh"
+    mode: '0755'
+    owner: "{{ project_user }}"
+    group: "{{ project_user }}" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)